docs/ci/ci.md

```diff
@@ -9,23 +9,30 @@ local workflow that controls the test run is located in
 [ovn-kubernetes/.github/workflows/test.yml](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml).
 
 The following tasks are performed:
+
 - Build OVN-Kubernetes
 - Check out the Kubernetes source tree and compiles some dependencies
 - Install KIND
 - Run a matrix of End-To-End Tests using KIND
 
-The following sections should help you understand (and if needed modify) the set of tests that run and how to run these tests locally.
+The full matrix of e2e tests found [here](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml)
```

Comment:
nit: The line 7-9 above seem to say the same thing. I understand the new bit you have added is the fact we run this twice daily, but now we are repeating the first part in a very short duration of time linking to the same matrix :)

Reply:
you are right. I'll clean this up in my next PR for this work.

Reply:
@tssurya , better late than never, but added this cleanup in a new PR [here](https://github.com/ovn-org/ovn-kubernetes/pull/4746/commits/d3b1a09fae73f2490df671f5f142ba7022a1ae99)